iosHackthonApp/Modules/Features/MealList/Tests/LinuxMain.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
+
+import MealListTests
+
+var tests = [XCTestCaseEntry]()
```

Comment:
This linux stuff isn't needed, unfortunately Xcode creates it automatically so I'll have to just delete